pkg/web_css/lib/src/_list.scss

```diff
@@ -128,15 +128,26 @@
   }
 }
 
+.thumbnail-image {
+  border: solid 1px #d5d4d4;
+}
+
 .collections-icon {
-  flex-grow: 0;
-  flex-shrink: 0;
-  flex-basis: 30px;
+  height: 30px;
   width: 30px;
-  align-self: flex-end;
-  margin-left: -30px;
   opacity: 70%;
   background-color: #aeaeae;
+  margin-top: auto;
```

Comment:
maybe a `position: relative; left: -30px` could be used here, but I'm fine with this too

Reply:
I cannot make it work with that, so I'll keep this since you are okay with this too.